docs/migration/6_0/html.md

```diff
@@ -83,3 +83,6 @@
 
 - Removed empty option(`ng-option`) from the title code selector(`ng-select`).
 
+### UserRegistrationFormComponent
+
+- Added id attribute to `ng-select` to generate `aria-controls` for accessibility improvements.
```

Comment:
Oh shoot, I may have missed something, but it's relatively small.
```suggestion
- Added `id` attribute to `ng-select` to generate `aria-controls` for accessibility improvements.
```

Reply:
^^ done